src/main.js

```diff
@@ -277,13 +279,79 @@ let load = loader({
     },
   },
 
+  s3Controller: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      return new remoteS3.Controller({
+        region: cfg.app.blobArtifactRegion,
+        accessKeyId: cfg.aws.accessKeyId,
+        secretAccessKey: cfg.aws.secretAccessKey,
+      });
+    },
+  },
+
+  // This is a dependency to ease the creation of buckets for, e.g. testing
+  createBuckets: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      let s3 = new aws.S3({
+        region: cfg.app.blobArtifactRegion,
+        accessKeyId: cfg.aws.accessKeyId,
+        secretAccessKey: cfg.aws.secretAccessKey,
+      });
+
+      // First, we'll create a public bucket
+      let publicConfig = {
+        Bucket: cfg.app.publicBlobArtifactBucket,
+        ACL: 'public-read',
+      };
+      
+      // Next, the private one
+      let privateConfig = {
+        Bucket: cfg.app.privateBlobArtifactBucket,
+        ACL: 'private',
+      };
+
+      if (cfg.app.blobArtifactRegion !== 'us-east-1') {
+        let x = (y) => {
+          y.CreateBucketConfiguration = {
+            LocationConstraint: cfg.app.blobArtifactRegion
+          };
+        }
+        x(publicConfig);
+        x(privateConfig);
```

Comment:
Finally, is it perhaps not better to create these buckets and configure them outside the queue code..
There are pros/cons... Amongst the pros is that if we configure it outside the queue code, then the queue needs a lot fewer privileges.. So we can be more strict in the AIM policy we grant the queue.